tailwind.config.ts

```diff
@@ -173,6 +173,10 @@ export const DreamPipColors = {
   },
 };
 
+// export const DreamPipColors = {
+//   white: '#ffffff'
+// };
```

Comment:
https://www.mixcloud.com/livetwentyfour7/sunday-sessions-with-jeff-white-and-special-guest-brian-borden-march-15-2022/